.changes/unreleased/Under the Hood-20220427-112127.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Under the Hood
+body: Mypy -> 0.942 + fixed import logic to allow for full mypy coverage
+time: 2022-04-27T11:21:27.499359-05:00
+custom:
+  Author: iknox-fa
+  Issue: "4805, 4089"
```

Comment:
@emmyoop Can I put two issues in this field?  This PR does legit address them both but I don't know if changie knows how to handle it...

Reply:
if not you can always do two change log entries. that's been done before.

Reply:
In the current state you cannot link multiple issues.  If you see this as valuable, we can definitely add it though.  We control how the fields end up being generated.  I recently made an update to allow multiple contributor names.